main.tf

```diff
@@ -57,6 +57,7 @@ module "access_logs" {
   noncurrent_version_transition_days = var.noncurrent_version_transition_days
   standard_transition_days           = var.standard_transition_days
   force_destroy                      = var.alb_access_logs_s3_bucket_force_destroy
+  context                            = module.this.context
```

Comment:
@jwstric2 since you added the `context` argument, can you remove the `name`, `namespace`, `stage`, `environment`, `delimiter`, and `tags`? `context` will replace all those values so they're not needed. Obviously, let's keep `attributes`.

Reply:
will do, I forgot to ask if those should be removed when opening up this PR.